src/components/form-textarea/README.md

```diff
@@ -162,7 +162,7 @@ explicitly set, `<b-form-textarea>` will automatically set the `aria-invalid` at
 
 ## Formatter support
 
-Refer to the (`<b-form-input>`)[../form-input] documentation regarding usage of the optional
+Refer to the [`<b-form-input>`](../form-input) documentation regarding usage of the optional
```

Comment:
Please also change the link to "/docs/components/form-input".

Reply:
Done